### ISSUE TITLE
[FEAT]: Publish Docker Image

### ISSUE THREAD

**Issue by SchoolGuy** [sapcc/redfish-exporter#11]
~~Would you be open to accepting a contribution that offers this exporter as a Docker Image?~~

~~The idea here for me is that the monitoring stack that I am currently building is containerized and this exporter is one of the few missing pieces for it.~~

~~If there are constraints due to SAP company policy, I am willing to follow those (for example chosen base-image etc).~~

Publish the image on Docker Hub please.

**Comment by baonq-me** [comment]
@fwiesel checkout my forked repo
https://github.com/baonq-me/redfish-exporter

Usage

```
docker run -d --restart=always -e USERNAME=redfish -e PASSWORD=xxxxxxxxx -p 9210:9210 quocbao747/redfish-exporter:1.0-beta
```

Run

```
curl "http://localhost:9210/performance?target=bmc.example.com"
```

**Comment by BerndKue** [comment]
I will not provide the docker image for you. We are working with our own internal docker registry. 
There is a dockerfile in the repo than can easily be used to create your own docker image.
Did you also see the comment above from @baonq-me?

**Comment by SchoolGuy** [comment]
Yes I saw but if I depend on the image of an individual, I can (for such a simple image) build it myself. His work is appreciated but since this would be running in production, I would rather prefer that the maintenance burden is shared on a group like sapcc.

Furthermore when I would be able to use the image, it would be possible for me to test with more servers and add support for more features if needed. Since his repository is a fork of yours, there would be an additional level of indirection that I'd like to avoid.
